jalib/jalloc.h

```diff
@@ -28,7 +28,8 @@
 #include <string.h>
 #include <unistd.h>
 
-#define JALIB_ALLOCATOR
+// FIXME: #ifndef __riscv__
+//        #define JALIB_ALLOCATOR
```

Comment:
The above two lines do this for all CPUs.  That can't be correct.  Further, the latest 'main' should now support JALIB, and you no longer need to do this.